README.md

```diff
@@ -4,19 +4,133 @@
-| :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
-| <small><a href="https://etherscan.io/token/0x3231cb76718cdef2155fc47b5286d82e6eda273f">![EUR](docs/0x3231cb76718cdef2155fc47b5286d82e6eda273f.png)</a><br /><a href="https://manager.ens.domains/name/eur.monerium.eth">eur.monerium.eth</a></small> | <small><a href="https://etherscan.io/token/0x7ba92741bf2a568abc6f1d3413c58c6e0244f8fd">![GBP](docs/0x7ba92741bf2a568abc6f1d3413c58c6e0244f8fd.png)</a><br /><a href="https://manager.ens.domains/name/gbp.monerium.eth">gbp.monerium.eth</a> | <small><a href="https://etherscan.io/token/0xc642549743a93674cf38d6431f75d6443f88e3e2">![ISK address](docs/0xc642549743a93674cf38d6431f75d6443f88e3e2.png)</a><br /><a href="https://manager.ens.domains/name/isk.monerium.eth">isk.monerium.eth</a></small> | <small><a href="https://etherscan.io/token/0xbc5142e0cc5eb16b47c63b0f033d4c2480853a52">![USD address](docs/0xbc5142e0cc5eb16b47c63b0f033d4c2480853a52.png)</a><br /><a href="https://manager.ens.domains/name/usd.monerium.eth">usd.monerium.eth</a></small> |
+Monerium e-money tokens [v1.0.3](https://github.com/monerium/smart-contracts/releases/tag/v1.0.3) are live on Ethereum mainnet and are [ERC20](https://eips.ethereum.org/EIPS/eip-20) and [ERC677](https://github.com/ethereum/EIPs/issues/677) compliant.
+
+### Euro / EUR
+
```

Comment:
Cosmetics again:

Logos and QR codes should be 1/2 of this size and aligned center in the cell.

Reply:
I'll make it smaller, but I disagree with center alignment.